src/components/datetime/DateTimeController.cpp

```diff
@@ -75,6 +75,22 @@ void DateTime::UpdateTime(uint32_t systickCounter) {
   minute = time.minutes().count();
   second = time.seconds().count();
 
+  if(minute == 0 and not isHourAlreadyNotified) {
```

Comment:
Needs formatting with `clang-format`.

Conditional does not match the syntax used in the rest of the code.